lib/middleware/errors.js

```
@@ -24,14 +25,18 @@ function _applicationError(err, req, res, next) {
 function _authenticationError(err, req, res, next) {
   logger.log('warn', {'Unauthorized': 'Invalid user credentials'});
   logger.log('warn', {'Error stack': err.stack});
-  res.redirect(uriConstructor.redirectUrl(req.originalUrl));
+
+  return fetch(config.LOGOUT_URL, { headers: req.headers })
```

So just swallow the error?